cli/src/main.rs

```diff
@@ -58,8 +58,7 @@ fn main() {
         // Register a client.
         // There's no persistence. So once the client app stops you have to
         // register a new client.
-        if op.starts_with("register ") {
-            let client_name = &op[9..];
+        if let Some(client_name) = op.strip_prefix("register ") {
```

Comment:
ugh, unrelated changes... please move them to a separate PR.

Reply:
I removed clippy warnings that were not caught before. This is just cleanup to make the CI happy since I touched that file.